app/scripts/controllers/groups/EditGroupController.js

```diff
@@ -107,6 +107,10 @@
                     externalId: data.externalId,
                     staffId: data.staffId
                 };
+                if (data.timeline.submittedOnDate) {
```

Comment:
data.timeline is not defined in this particular response. So the data.timeline will be undefined.